Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -39,6 +39,18 @@ class RouterSpec: QuickSpec {
                     expect(router.process(request: aHeadRequest)).to(equal(formatter.format(httpResponse: expectedResponse)))
                 }
             }
+
+            context("when receive a request to an unknown URL") {
+                it("returns an response of 404 Not Found status") {
```

Comment:
Do you need to force unwrap the optional here? Can you use optional chaining instead?

https://docs.swift.org/swift-book/LanguageGuide/OptionalChaining.html

Reply:
Because all URLs used in for the server are pre-defined and not nil, it's safe to use ! in this particular case. I think the use of a fall back doesn't make sense here.